app/controllers/ControllerHelper.scala

```diff
@@ -46,7 +46,8 @@ class ControllerHelper @Inject() (
     with Logging {
 
   private def indAlreadyRegistered(implicit request: DataRequest[AnyContent]) =
-    request.affinityGroup == AffinityGroup.Individual ||
+    request.userAnswers.get(ReporterTypePage).exists(ReporterType.nonOrgReporterTypes.contains) ||
```

Comment:
Only gripe is this boolean - could you make it clear what this is? like **isNotOrganisation** (i think thats what it's saying?)

`request.userAnswers.get(ReporterTypePage).exists(ReporterType.nonOrgReporterTypes.contains)`